### DISCUSSION TITLE
Why does the same turtlebot.usd with different versions cause different displays, thereby affecting the mapping?

### DISCUSSION THREAD

**Discussion by zhaoyaping111** [StanfordVL/BEHAVIOR-1K#1096] [General]
isaac sim 2022.2.0         isaac sim 4.1

**Comment by hang-yin** [comment]
Hi @zhaoyaping111 , what specifically do you mean by display and mapping? Do you mean that the same turtlebot usd look visually different across different versions of Isaac sim?
